frontend/src/components/Inquiries/InquiriesTable.tsx

```diff
@@ -1,86 +1,102 @@
+    /* eslint-disable */
+    const parsedDate = dayjs.utc(date)
+    const userTimezone = dayjs.tz.guess()
+    return parsedDate.isValid()
+      ? parsedDate.tz(userTimezone).format("MMM DD, YYYY hh:mm A")
+      : invalidDateMessage
+    /* eslint-enable */
+  }
 
-// Format date to user's timezone
-// ex. Sep 17, 2024 14:13 PM
-function formatDate(date: Date): string {
-    try {
-        // eslint-disable-next-line
-        return dayjs.utc(date).tz(userTimezone).format("MMM DD, YYYY hh:mm A");
-    } catch (error) {
-        console.error("Error formatting date:", error);
-        return new Date(date).toISOString();
+  function getInquiriesQueryOptions() {
+    return {
+      queryKey: ["inquiries"],
+      queryFn: () => InquiriesService.readInquiries(),
     }
-}
+  }
 
-function InquiriesTable() {
-    const { data: inquiries, isPending } = useQuery({
-        ...getInquiriesQueryOptions(),
-    });
+  const { data: inquiries, isPending } = useQuery({
+    ...getInquiriesQueryOptions(),
+  })
 
-    // Sort inquries from Newest to oldest
-    const sortedInquiries = inquiries?.data.sort((a, b) => {
-        return new Date(b.created_at).getTime() - new Date(a.created_at).getTime();
-    });
+  // Sort inquries from Newest to oldest
+  const sortedInquiries = inquiries?.data.sort((a, b) => {
+    return new Date(b.created_at).getTime() - new Date(a.created_at).getTime()
+  })
 
-    return (
-        <>
-            <TableContainer>
-                <Table>
-                    <Thead>
-                        <Tr>
-                            <Th>Text</Th>
-                            <Th>Created At</Th>
-                        </Tr>
-                    </Thead>
-                    {isPending ? (
-                        <Tbody>
-                            <Tr>
-                                {new Array(3).fill(null).map((_, index) => (
-                                    <Td key={index}>
-                                        <SkeletonText noOfLines={1} />
-                                    </Td>
-                                ))}
-                            </Tr>
-                        </Tbody>
-                    ) : (
-                        <Tbody>
-                            {sortedInquiries?.map((inquiry) => (
-                                <Tr
-                                    key={inquiry.id}
-                                    onClick={() => {
-                                        console.log(inquiry);
-                                    }}
-                                >
-                                    <Td>{inquiry.text}</Td>
-                                    <Td>{formatDate(inquiry.created_at)}</Td>
-                                </Tr>
-                            ))}
-                        </Tbody>
-                    )}
-                </Table>
-            </TableContainer>
-        </>
-    );
+  return (
+    <>
+      <TableContainer>
```

Comment:
Nit, this would require a pretty major refactor but we should make a generic table component that can be shared. It looks like we have duplicated the table in `routes/_layout/admin`, `routes/_layout/items',  and `components/Inquiries/InquiriesTable`.

Reply:
I was thinking about this when I was making `InquiriesTable` component. We will eventually remove `routes/_layout/admin` and `routes/_layout/items` from our app since they don't fit with our app's feature. As we add more feature pages, we can definitely refactor it to use generic table component. 

I also created `formatDate` function inside `InquiriesTable` component. In the future, we'll want this function as a utility function if other pages need to display date.